setup.cfg

```diff
@@ -7,3 +7,30 @@ testpaths = onnx/test onnx/examples
 
 [metadata]
 license-file = LICENSE
+
+[mypy]
+follow-imports = silent  # TODO first make this 'silent', then remove entirely
+mypy_path = stubs:third_party/caffe2:third_party/onnx
```

Comment:
caffe2 shouldn't be here probably

Reply:
Yes, this is still WIP. I need to create this PR to trigger travis builds because I can't push branches directly to the repo.

Reply:
Note: removed caffe2. Numpy has too complex type relations for correct and comprehensive type stubs. I added some good-enough ones to the onnx-coreml repo and will add them here too later, but I want this PR to just be the minimal set of code changes needed to make mypy pass.